chords/in_christ_alone.onsong

```diff
@@ -2,6 +2,10 @@
 {artist:  Keith Getty / Stuart Townsend}
 {key: D}
 
+intro:
+|[A].   .   .   |[G].   .   .   |[D].   .   [G].   |[D].   .   .   |
```

Comment:
I don't know if we want to use this instrumental bit between verses or not, but if we do, then we might as well include it!